modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPOrganization.macro

```diff
@@ -0,0 +1,148 @@
+	}
+
+	@summary = "Fill in all required fields on the Organization Form"
+	macro fillOrganizationForm {
+		WaitForPageLoad();
+
+		if (isSet(organizationName)) {
+			Type(
+				key_fieldLabel = "Organization Name",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${organizationName});
+		}
+
+		if (isSet(organizationSite)) {
+			Type(
+				key_fieldLabel = "Organization Site/Social Media Link",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${organizationSite});
+		}
+
+		if (isSet(smallDescription)) {
+			Type(
+				key_fieldLabel = "Small Description",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${smallDescription});
+		}
+
+		if (isSet(street)) {
+			Type(
+				key_fieldLabel = "Street",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${street});
+		}
+
+		if (isSet(city)) {
+			Type(
+				key_fieldLabel = "City",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${city});
+		}
+
+		if (isSet(state)) {
+			Type(
+				key_fieldLabel = "State",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${state});
+		}
+
+		if (isSet(country)) {
+			Type(
+				key_fieldLabel = "Country",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${country});
+		}
+
+		if (isSet(zipCode)) {
+			Type(
+				key_fieldLabel = "Zip",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${zipCode});
+		}
+
+		if (isSet(taxId)) {
+			Type(
+				key_fieldLabel = "Tax Id",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${taxId});
+		}
+
+		if (isSet(contactName)) {
+			Type(
+				key_fieldLabel = "Contact Name",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${contactName});
+		}
+
+		if (isSet(contactEmail)) {
+			Type(
+				key_fieldLabel = "Contact Email",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${contactEmail});
+		}
+
+		if (isSet(contactPhone)) {
+			Type(
+				key_fieldLabel = "Contact Phone",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${contactPhone});
+		}
+
+		if (isSet(bankingInfo)) {
+			Type(
+				key_fieldLabel = "Banking Info",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = ${bankingInfo});
+		}
+
+		ScrollWebElementIntoView(locator1 = "Button#SUBMIT");
+
+		Button.click(button = "Submit");
+
+		if (isSet(createRequest)) {
+			Button.click(button = "Create a Request");
+		}
+		else {
+			Button.click(button = "Done");
+		}
+	}
+
+	@summary = "Add a new organization"
+	macro goToCreateOrganization {
+		EVPNavigator.goToNavItem(navItem = "Organization");
+
+		Button.click(button = "Create an Organization");
+	}
+
+	@summary = "Click kebab menu in organization page"
+	macro kebabMenuOrganization {
+		Click(
+			key_name = ${organizationName},
+			locator1 = "EVPUtils#KEBAB_MENU_OPTION_ORGANIZATION");
+	}
+
+	@summary = "Option kebab menu is not present"
+	macro viewOptionNotPresentInKebabMenu {
+		MenuItem.viewNotPresent(menuItem = ${menuItem});
+	}
```

Comment:
Hi @joelrocha68, as `MenuItem.viewNotPresent(menuItem = ${menuItem});` already is a macro, I think that you shouldn't create another macro for it. What do you think about just use the macro that already exists?

Reply:
Done